src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/VideoAppointmentRepository.kt

```diff
@@ -42,4 +42,21 @@ interface VideoAppointmentRepository : ReadOnlyRepository<VideoAppointment, Long
     startTime: LocalTime,
     endTime: LocalTime,
   ): VideoAppointment?
+
+  @Query(
+    value = """
+      FROM VideoAppointment va 
+      WHERE va.prisonCode = :prisonCode
+      AND va.prisonerNumber = :prisonerNumber
+      AND va.appointmentDate = :appointmentDate
+      AND va.startTime = :startTime
+      AND va.statusCode = 'ACTIVE'
+    """,
```

Comment:
Are appointments changed to CANCELLED when err.. cancelled?

Reply:
Err yes.

This is looking up active bookings though in preparation to be cancelled so not sure what your point is ...

Reply:
Good point, no 🤦